src/components/ebay-carousel/style.less

```diff
@@ -67,6 +69,35 @@
         }
     }
 
+    &__play, &__pause {
+        z-index: 2;
+        bottom: 16px; // accounts for padding.
+        right: 0;
+        margin-right: 19px;
+        margin-bottom: 19px;
+        position: absolute;
+        background: 0;
```

Comment:
Is this needed with `background-color` below?

Reply:
This was just c&p'd from homepage. Will see if we actually need it.